api/models.py

```diff
@@ -163,7 +163,24 @@ class Revision(BaseModel):
     )
 
 
+class KernelBuild(BaseModel):
+    """Kernel build data"""
+    arch: str = Field(
+        description="CPU architecture as per the kernel's convention"
+    )
+    defconfig: str = Field(
+        description="Name of the base defconfig file used for the build"
+    )
+    fragments: list = Field(
+        description="List of the config fragment names for the build"
+    )
+    compiler: str = Field(
+        description="Name of the compiler used for the build"
+    )
+
+
```

Comment:
I guess you are going to drop this. Right?

Reply:
Yes it just needs a rebase